codegenerator/cli/npm/envio/index.d.ts

```diff
@@ -1 +1,110 @@
-export { logger as Logger } from "./src/Envio.gen";
+export type {
+  logger as Logger,
+  effect as Effect,
+  effectContext as EffectContext,
+  effectArgs as EffectArgs,
+  effectOptions as EffectOptoins,
```

Comment:
Might have propagating issues changing the name here...
```suggestion
  effectOptions as EffectOptions,
```